### PR TITLE
feat(release): update c24cd1d

### PR DESCRIPTION
**Pull request by dtfiedler**

### CODE REVIEW

**Review by fedellen — APPROVED** [review]
LGTM!

File: src/migrations/20231106203141_nullable_content_type.ts [hunk 1]
```diff
@@ -14,22 +14,14 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import axios, { AxiosResponse } from "axios";
+import { Knex } from "knex";
 
-import { TransactionId } from "../../types/types";
-import { ContractReader } from "./contractOracle";
+import { NullableContentTypeMigrator } from "../../src/arch/db/migrator";
```
Comment (MichaelBuhler) [inline src/migrations/20231106203141_nullable_content_type.ts:19]: This import statement reach up out of the `src` dir to the project root and then back down into the `src` dir. This presents problems after TypeScript compilation, when the compiled JavaScript code in the `/lib` dir attempts to import TypeScript code from the `/src` dir.

Comment (MichaelBuhler) [reply src/migrations/20231106203141_nullable_content_type.ts:19]: Causes #4.

**Review by MichaelBuhler — COMMENTED** [review]
Causes #4.

**Pull request merged**